x-pack/plugins/ml/public/application/components/data_grid/use_column_chart.tsx

```diff
@@ -231,11 +231,13 @@ export const useColumnChart = (
   if (isOrdinalChartData(chartData)) {
     data = chartData.data.map((d: OrdinalDataItem) => ({
       ...d,
+      key_as_string: d.key_as_string || d.key,
```

Comment:
if `d.key_as_string ` could be `undefined` this should probably be a `??` check.
also, should `key_as_string` be optional in the interface?

Reply:
Yes, `d.key_as_string` is not defined in some items this point so we are checking for `undefined`. Great suggestion for updating to use `??` check.

As `key_as_string` will now always be defined (since we're setting it as `key` when it isn't) I left it as required in the interface.